README.md

```diff
@@ -90,14 +90,15 @@ To choose from three configuration settings, install the [`eslint-config-lwc`](h
 
 ### Best practices
 
-| Rule ID                                                                  | Description                                               | Fixable |
-| ------------------------------------------------------------------------ | --------------------------------------------------------- | ------- |
-| [lwc/no-async-operation](./docs/rules/no-async-operation.md)             | restrict usage of async operations                        |         |
-| [lwc/no-dupe-class-members](./docs/rules/no-dupe-class-members.md)       | disallow duplicate class members                          |         |
-| [lwc/no-inner-html](./docs/rules/no-inner-html.md)                       | disallow usage of `innerHTML`                             |         |
-| [lwc/no-template-children](./docs/rules/no-template-children.md)         | prevent accessing the immediate children of this.template |         |
-| [lwc/no-leaky-event-listeners](./docs/rules/no-leaky-event-listeners.md) | prevent event listeners from leaking memory               |         |
-| [lwc/prefer-custom-event](./docs/rules/prefer-custom-event.md)           | suggest usage of `CustomEvent` over `Event` constructor   |         |
+| Rule ID                                                                                                            | Description                                                | Fixable |
+| ------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------- | ------- |
+| [lwc/no-async-operation](./docs/rules/no-async-operation.md)                                                       | restrict usage of async operations                         |         |
+| [lwc/no-dupe-class-members](./docs/rules/no-dupe-class-members.md)                                                 | disallow duplicate class members                           |         |
+| [lwc/no-inner-html](./docs/rules/no-inner-html.md)                                                                 | disallow usage of `innerHTML`                              |         |
+| [lwc/no-template-children](./docs/rules/no-template-children.md)                                                   | prevent accessing the immediate children of this.template  |         |
+| [lwc/no-leaky-event-listeners](./docs/rules/no-leaky-event-listeners.md)                                           | prevent event listeners from leaking memory                |         |
+| [lwc/prefer-custom-event](./docs/rules/prefer-custom-event.md)                                                     | suggest usage of `CustomEvent` over `Event` constructor    |         |
+| [lwc/no-unsupported-node-api-in-ssrable-components](./docs/rules/no-unsupported-node-api-in-ssrable-components.md) | disallow unsupported Node API calls in SSR-able components |         |
```

Comment:
```suggestion
| [lwc/ssr/no-unsupported-node-api](./docs/rules/no-unsupported-node-api.md) | disallow unsupported Node API calls in SSR-able components |         |
```

Feedback from @wjhsf 

In the eslint plugin, there’s currently no consistency in the SSR rule names:
- `lwc/no-restricted-browser-globals-during-ssr`
- `lwc/no-unsupported-ssr-properties`
- `lwc/no-node-env-in-ssr`
- `lwc/no-unsuppoted-node-api-in-ssrable-components`
- `lwc/no-unguarded-async-event-in-ssr`
- `lwc/no-form-factor-in-ssrable-components`
- `lwc/no-static-imports-of-user-specific-scoped-modules-in-ssrable-components`

To have a more consistent naming convention (and terser names), please change all of them to the format `lwc/ssr/*`.

For the few older ones that have already been published, we could either do a rename and release a major, or we could keep the old ones as aliases, but mark them as deprecated.

Reply:
Sure. I'll rename them.

Reply:
I renamed all references to the rule by prepending 'ssr-'.